lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -2688,6 +2722,32 @@ class ConvertAtenPoolingBaseOp : public OpConversionPattern<AtenOpT> {
                                           : nhwcToNchw4DTransposeDims);
   }
 
+  Type getOutputType(RankedTensorType inputTy,
```

Comment:
Since this function is defined in the base class for pooling, it should work for all the dependents; however, from what I can tell, this function will not work for `AtenAdaptivePoolingOp`. If this function can't be made to work with that op, then it should be placed outside this base class